docs/crr-to-aws-s3.md

```diff
@@ -75,3 +75,340 @@ env_replication_endpoints:
+to Amazon Web Services S3 using the Backbeat replication engine.
+
+### Half-Automated Example
+
+This example demonstrates how to set up replication using a built-in command
+line tool provided by Backbeat.
+
+1. Create the target bucket:
+
+    ```
+    aws s3api create-bucket \
+    --bucket <target-bucket> \
+    --profile <target-profile>
+    ```
+
+2. Enable versioning on the target bucket:
+
+    ```
+    aws s3api put-bucket-versioning \
+    --bucket <target-bucket> \
+    --versioning-configuration Status=Enabled \
+    --profile <target-profile>
+    ```
+
+3. Generate account credentials for the source site using the command:
+
+    ```
+    ansible-playbook -i env/client-template/inventory \
+    tooling-playbooks/generate-account-access-key.yml
+    ```
+
+4. At the source, log in to the docker shell of a Backbeat container:
+
+    ```
+    docker exec -it <backbeat-container-name> bash
+    ```
+
+    Note that on a full deployment the container name for Backbeat would be
+    'scality-backbeat-1' while with a light-local deployment the name would
+    be 'scality-backbeat-127.0.0.2-1'.
+
+5. Store the credentials generated in step 3 as the profile `<source-profile>`:
+
+    ```
+    mkdir ~/.aws
+    vi ~/.aws/credentials
+    ```
+
+    The completed file should be similar to the following:
+
+    ```
+    [<source-profile>]
+    aws_access_key_id = <access-key>
+    aws_secret_access_key = <secret-key>
+    ```
+
+6. Set up replication on your buckets:
+
+    ```
+    node ~/replication/backbeat/bin/setupReplication.js setup \
+    --source-bucket <source-bucket> \
+    --source-profile <source-profile> \
+    --target-bucket <target-bucket> \
+    --site-name <target-site-name>
+    ```
+
+### Manual Example
+
+This example demonstrates how to set up replication manually using the AWS cli.
+
+1. Create source and target buckets:
+
+    ```
+    aws s3api create-bucket \
+    --bucket <source-bucket> \
+    --endpoint http://<source> \
+    --profile <source-profile>
+    ```
+
+    ```
+    aws s3api create-bucket \
+    --bucket <target-bucket> \
+    --profile <target-profile>
+    ```
+
+2. Enable bucket versioning on each bucket:
+
+    ```
+    aws s3api put-bucket-versioning \
+    --bucket <source-bucket> \
+    --versioning-configuration Status=Enabled \
+    --endpoint http://<source> \
+    --profile <source-profile>
+    ```
+
+    ```
+    aws s3api put-bucket-versioning \
+    --bucket <target-bucket> \
+    --versioning-configuration Status=Enabled \
+    --profile <target-profile>
+    ```
+
+    Verify that bucket versioning is enabled on each bucket:
+
+    ```
+    aws s3api get-bucket-versioning \
+    --bucket <source-bucket> \
+    --endpoint http://<source> \
+    --profile <source-profile>
+    ```
+
+    ```
+    aws s3api get-bucket-versioning \
+    --bucket <target-bucket> \
+    --profile <target-profile>
+    ```
+
+3. Establish a bucket policy to allow the Backbeat replication engine to
```

Comment:
Let's not call it a bucket policy since bucket policies are a whole different game. `IAM resource policy` would be correct.
```
Establish an IAM resource policy on the source bucket to allow the Backbeat to
execute the required actions.
```